src/check/expr.rs

```diff
@@ -26,34 +26,40 @@ pub fn check_infer(index: u32, tcs: TCS, expression: Expression) -> TCM<Value> {
             let context = tcs.context.clone();
             let inferred = Either::Left(check_infer(index, tcs, *expression)?);
             map.insert(name, Box::new(inferred));
-            Ok(Value::Sum(GenericCaseTree::boxing(map, context)))
+            Ok(Value::Sum(GenericCaseTree::boxing(map, context), 0))
         }
         Pair(left, right) => {
             let left = check_infer(index, tcs_borrow!(tcs), *left)?;
             let right = check_infer(index, tcs_borrow!(tcs), *right)?;
             let right = Closure::Value(Box::new(right));
-            Ok(Value::Sigma(Box::new(left), right))
+            Ok(Value::Sigma(Box::new(left), right, 0))
```

Comment:
This is a magic number. This is supposed to be the the return value of something like `Value::level()`